internal/pkg/gateway/config.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package gateway
+
+// Configuration defaults
+var (
```

Comment:
Please replace the package level var with a function that returns an instance containing the default options.